### PR TITLE
Add tooltips for every* unfinished block/tool

### PR DESCRIPTION
**Pull request by jusvit**
I added the "WIP Coming Soon" for a vast majority of the unfinished blocks. This should prevent any questions arising from the users when they try and use unfinished blocks.

### CODE REVIEW

File: src/main/java/techreborn/blocks/BlockNuke.java [hunk 1]
```diff
@@ -143,4 +147,10 @@ protected BlockStateContainer createBlockState() {
 		return new BlockStateContainer(this, OVERLAY);
 	}
 
+	@Override
+	public void addInformation(ItemStack stack, EntityPlayer playerIn, List<String> tooltip, boolean advanced) {
```
Comment (modmuss50) [inline src/main/java/techreborn/blocks/BlockNuke.java]: This works just fine. It has no recipe as it has no sane need in survival.

File: src/main/java/techreborn/blocks/BlockPlayerDetector.java [hunk 2]
```diff
@@ -178,4 +178,11 @@ public IBlockState getStateFromMeta(int meta) {
 	public int getMetaFromState(IBlockState state) {
 		return typeNamesList.indexOf(state.getValue(TYPE));
 	}
+
+	@Override
+	public void addInformation(ItemStack stack, EntityPlayer playerIn, List<String> tooltip, boolean advanced) {
```
Comment (modmuss50) [inline src/main/java/techreborn/blocks/BlockPlayerDetector.java]: Same again, works but needs a recipe.

File: src/main/java/techreborn/blocks/machine/BlockAssemblingMachine.java [hunk 3]
```diff
@@ -84,4 +88,11 @@ public String getTop() {
 	public String getBottom() {
 		return this.prefix + "assembling_machine_top";
 	}
+
+	@Override
+	public void addInformation(ItemStack stack, EntityPlayer playerIn, List<String> tooltip, boolean advanced) {
```
Comment (modmuss50) [inline src/main/java/techreborn/blocks/machine/BlockAssemblingMachine.java]: This is not supposed to have a recipe or any recipes

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Either need recipes or the WIP tag removing off the ones I've mentioned

File: src/main/java/techreborn/blocks/BlockPlayerDetector.java (hunk 2 above)
Comment (jusvit) [reply src/main/java/techreborn/blocks/BlockPlayerDetector.java]: The detect player (All) works but the other two modes do not they act the same as the (All) block

File: src/main/java/techreborn/blocks/machine/BlockAssemblingMachine.java (hunk 3 above)
Comment (jusvit) [reply src/main/java/techreborn/blocks/machine/BlockAssemblingMachine.java]: But it's still a WIP though?

Comment (modmuss50) [reply src/main/java/techreborn/blocks/machine/BlockAssemblingMachine.java]: No, it's desgined for pack devs to use.

File: src/main/java/techreborn/blocks/BlockPlayerDetector.java (hunk 2 above)
Comment (modmuss50) [reply src/main/java/techreborn/blocks/BlockPlayerDetector.java]: That's a bug then, and should be fixed.

**Comment by jusvit** [comment]
Did all the requested changed, ready for merge.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**